apiary.apib

```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
         
 + Response 403 (application/json)        
 
-        {status:0 , msg:"Only organization administrators can update referrals."}           
+        {status:0 , msg:"Only organization administrators can update referrals."}        
+        
+### Get referral system subscribers [/organizations/1/referral-system/] [GET]        
+
++ Response 200 (application/json)
+        {
+           "Potential": [
```

Comment:
@hakob9315 let's keep it in solid style, so rename it please `Potential` -> `potential` . Same for `Confirmed` -> `confirmed`

Reply:
ok